### PR TITLE
{bio}[GCC/9.3.0] Snippy v4.6.0 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by jfgrimm**
(created using `eb --new-pr`)
Requires #12841, #12856, #12859, #1288, #12892, #12898 and #12900

### CODE REVIEW

**Comment by jfgrimm** [comment]
@branfosj with freebayes approved recently, all deps should be covered now

File: easybuild/easyconfigs/s/Snippy/Snippy-4.6.0-GCC-9.3.0-Java-13-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+dependencies = [
+    ('Java', '13', '', True),
+    ('Python', '3.8.2'),
+    ('Perl', '5.30.2'),
+    ('BioPerl', '1.7.7'),
+    ('any2fasta', '0.4.2'),
+    ('BWA', '0.7.17'),
+    ('BCFtools', '1.10.2'),
+    ('BEDTools', '2.29.2'),
+    ('freebayes', '1.3.5', local_pyversuffix),
```
Comment (branfosj) [inline easybuild/easyconfigs/s/Snippy/Snippy-4.6.0-GCC-9.3.0-Java-13-Python-3.8.2.eb:32]:
```suggestion
    ('freebayes', '1.3.5', local_pyversuffix),
    ('gzip', '1.10'),
```

I have `Found gzip - /usr/bin/gzip` in the log so we should as `gzip` as a dependency.

File: easybuild/easyconfigs/s/Snippy/Snippy-4.6.0-GCC-9.3.0-Java-13-Python-3.8.2.eb [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    ('snp-sites', '2.5.1'),
+    ('snpEff', '5.0', versionsuffix),
+    ('vcflib', '1.0.2', local_pyversuffix),
+    ('vt', '0.57721'),
+]
+
+skipsteps = ['configure', 'build']
+
+pretestopts = "cd test && "
+runtest = ''
```
Comment (branfosj) [inline easybuild/easyconfigs/s/Snippy/Snippy-4.6.0-GCC-9.3.0-Java-13-Python-3.8.2.eb:48]:
```suggestion
runtest = ' '
```

An empty string for `runtest` means that no tests are run.

**Review by branfosj — CHANGES_REQUESTED** [review]

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/69723b9c63b6b4cd2988b7a8c51dbe18 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/6ca3bff5f6b0da4c00c8124216fcf016 for a full test report.

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Going in, thanks @jfgrimm!

**Pull request merged**